larsim/EventGenerator/SingleGen_module.cc

```diff
@@ -447,11 +448,11 @@ namespace evgen {
     , fHistFileName(config().HistogramFile())
     , fPHist(config().PHist())
     , fThetaXzYzHist(config().ThetaXzYzHist())
-    , fEngine(createEngine(0))
+    , fEngine(art::ServiceHandle<rndm::NuRandomService>()
+                ->registerAndSeedEngine(createEngine(config().Seed()), "HepJamesRandom", "Seed"))
```

Comment:
```suggestion
    , fEngine(art::ServiceHandle<rndm::NuRandomService>()
                ->registerAndSeedEngine(createEngine(config().Seed()), "HepJamesRandom", "Seed"))
```
Note that `config().Seed()` in the `createEngine()` argument is always going to be overwritten, either by the same value again, or by the seed assigned to the engine by `NuRandomService`.
Whether having `config().Seed()` is more or less confusing than having `0` as argument is up to the reader (I prefer the latter because it's "clear" that it can't be the final setting, but again that's just me).

Reply:
Thanks for looking into this, @PetrilloAtWork.  I suggested that @mantheys use `InvalidSeed` as the default parameter value.